lineage/states/stateCommon.py

```diff
@@ -79,14 +79,19 @@ def gamma_estimator(gamma_obs: list[np.ndarray], time_cen: list[np.ndarray], gam
     if np.sum([np.sum(g) for g in gammas]) < 0.1:
         gammas = [np.ones(g.size) for g in gammas]
 
+    # make sure all negative observations are removed
+    gamma_obs_ = [gm[gamma_obs[i] >= 0] for i, gm in enumerate(gamma_obs)]
```

Comment:
Does the estimator need to do this step, if it's already done in the state distribution class? It seems like that can just pass the filtered variables...

Reply:
done.